tsk2/tsk2/Factory.swift

```diff
@@ -70,7 +124,7 @@ class Factory {
 }
 
 
-protocol DillerProtocol: AnyObject {
+protocol DelegateProtocol: AnyObject {
     
     func getCars() -> [CarProtocol]
     func addSedan(_ car: Sedan)
```

Comment:
один универсальный метод
```suggestion
    func factoryDidCreate(_ factory: Factory, car types: CarType, of amount: Int)
```

или


```suggestion
    func factoryDidCreate(_ factory: Factory, sedan cars: [Sedan])
    func factoryDidCreate(_ factory: Factory, pickUp cars: [PickUp])
    func factoryDidCreate(_ factory: Factory, sportCar cars: [SportCar])
```
то есть методы для всех типов авто